frontend/src/components/Admin/AdminShoutouts/AdminShoutouts.tsx

```diff
@@ -48,10 +48,16 @@ const AdminShoutouts: React.FC = () => {
   }, [earlyDate, lastDate, hide, updateShoutouts]);
 
   const ListTitle = (): JSX.Element => {
-    let title = 'All Shoutouts';
-    if (view === 'HIDDEN') title = 'Hidden Shoutouts';
-    if (view === 'PRESENT') title = '';
-    return <Header className={styles.formTitle} content={title}></Header>;
+    let title = `Showing ${displayShoutouts.length} of ${allShoutouts.length} shoutouts`;
+    if (view === 'HIDDEN') title = `Hidden Shoutouts (${displayShoutouts.length})`;
+    if (view === 'PRESENT') {
+      title = '';
+    }
+    return (
+      <div className={styles.formHeader}>
+        <Header className={styles.formTitle} content={title}></Header>
```

Comment:
There's no content in this `Header` component, so we can rewrite as `<Header className={styles.formTitle} content={title}/>`. Or `<Header className={styles.formTitle}>{title}</Header>`. But I wouldn't leave it as an empty component.